sklearn/manifold/spectral_embedding_.py

```diff
@@ -116,6 +117,17 @@ def _set_diag(laplacian, value):
     return laplacian
 
 
+def _eigen_sign_flip(u):
+    """Flipping sign of all the terms in every eigen vector
```

Comment:
This is a private function so docstrings are not *required* to be perfect... but It would be awesome if you could, in general, refer [this](https://github.com/scikit-learn/scikit-learn/wiki/How-to-write-docs).